src/Services/GTT/GTT.Api/SportsManagement/CreateSportsFunction.cs

```diff
@@ -15,55 +15,57 @@
 
 namespace GTT_API.ExcerciseGroupManagement
 {
-    public class CreateExerciseGroup
+    public class CreateSportsFunction
```

Comment:
Remove "Function" to make it consistent with other APIs

Reply:
I named this for the purpose of distinguishing between Azure Function and Command. Please help to review @ClearActionConsistent